pkg/osquery/tables/gsettings/metadata.go

```diff
@@ -0,0 +1,274 @@
+// +build !windows
+
+package gsettings
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	osquery "github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+	"github.com/pkg/errors"
+)
+
+type GsettingsMetadata struct {
+	client    *osquery.ExtensionManagerClient
+	logger    log.Logger
+	cmdRunner func(ctx context.Context, args []string, output *bytes.Buffer) error
+}
+
+// Metadata returns a table plugin for querying metadata about specific keys in
+// specific schemas
+func Metadata(client *osquery.ExtensionManagerClient, logger log.Logger) *table.Plugin {
+
+	columns := []table.ColumnDefinition{
+		// TODO: maybe need to add 'path' for relocatable schemas..
+		table.TextColumn("schema"),
+		table.TextColumn("key"),
+		table.TextColumn("description"),
+		table.TextColumn("type"),
+	}
+
+	t := &GsettingsMetadata{
+		client:    client,
+		logger:    logger,
+		cmdRunner: execGsettingsCommand,
```

Comment:
(nit) feels weird to use a different signature here, then above. It's basically the same functionality.

Reply:
agreed. this is the result of me copy/pasting from the gsettings code into the metadata, then iterating on the original and not applying the changes here.